pmlc/conversion/gpu/passes.td

```diff
@@ -0,0 +1,24 @@
+#ifndef __PMLC_CONVERSION_GPU_TO_VULKAN_PASSES__
+#define __PMLC_CONVERSION_GPU_TO_VULKAN_PASSES__
+
+include "mlir/Pass/PassBase.td"
+
+def ConvertGpuLaunchFuncToVulkanCalls : Pass<"pmlc-convert-gpu-to-vulkan", "mlir::ModuleOp"> {
+  let summary = "Convert gpu.launch_func op to Vulkan runtime calls";
+  let constructor = "pmlc::conversion::gpu::createConvertGpuLaunchFuncToVulkanCallsPass()";
+}
+
+#endif // __PMLC_CONVERSION_GPU_TO_VULKAN_PASSES__
+
+
+#ifndef __PMLC_CONVERSION_KERNEL_OUTLINING_PASSES__
+#define __PMLC_CONVERSION_KERNEL_OUTLINING_PASSES__
+
+include "mlir/Pass/PassBase.td"
```

Comment:
Remove these lines, this almost looks like multiple files were copy/pasted into one.

Reply:
Yes, I will remove these.